runtime/queries/yuck/highlights.scm

```diff
@@ -0,0 +1,66 @@
+(ERROR) @error
+
+(line_comment) @comment
+
+; keywords and symbols
+
+(keyword) @keyword
+(symbol) @tag
```

Comment:
```suggestion
(symbol) @string.special.symbol
```

I'm not sure if this is the same kind of symbol though: we currently use this scope for things like Erlang/Elixir atoms, Ruby symbols and Clojure keywords

Reply:
Yeah (symbol) highlighting could certainly be improved still, maybe some real keywords for things that are used a lot and are builtin like `defwidget`, `label` or `box`.

Hmm I'm not sure, it seems to be something like a DOM element definition, thus IMO `@tag` makes more sense here?

Reply:
Ah ok, if these are more DOM-like then let's keep `@tag` 👍